scripts/Follower.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python2
+import rospy
+import math
+from geometry_msgs.msg import PointStamped
+from ackermann_msgs.msg import AckermannDrive
+from std_msgs.msg import String
+from state_controller.msg import TwistLabeled
+from tf import TransformListener
+
+class Follower():
+    def __init__(self):
```

Comment:
Needs a class docstring to explain what this is (rosnode), what it does (follows a Point msg), and how to use it (publish a Point msg to a topic).

Reply:
Here's a pretty canonical example (doesn't need to be nearly this thorough) https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html